assert/almost_equals.ts

```diff
@@ -29,6 +29,7 @@ import { AssertionError } from "./assertion_error.ts";
  * default is one hundred thousandth of a percent of the expected value.
  * @param msg The optional message to include in the error.
  */
+// deno-lint-ignore deno-style-guide/exported-function-args-maximum
```

Comment:
Sidenote: bums me out a little that we don't follow this rule in `@std/assert` 😝